### ISSUE TITLE
Can't create any user or new domains with a user

### ISSUE THREAD

**Issue by yodahome** [modoboa/modoboa#1837]
# Impacted versions

* Modoboa: 1.14.0
* installer used: Yes
* Webserver: Apache

# Steps to reproduce / Current behavior

Logging in to the web Interface after a fresh installation (Ubuntu 18.04, did select NOT to install nginx, uwsgi and radicale because I already had an apache2 running), I could create a new domain without a user but cannot creata any additional accounts of any role, after entering the info, clicking next and trying to click submit, the app will not react.
Browser console gives this error:

VM2723:1 POST https://server.domain/admin/accounts/new/ 500 (Internal Server Error)
(anonymous) @ VM2723:1
send @ jquery.min.js:5
ajax @ jquery.min.js:5
post @ creation_wizard.js:41
submit @ creation_wizard.js:105
i @ jquery.min.js:3
dispatch @ jquery.min.js:3
v.handle @ jquery.min.js:3
VM2803:1 Uncaught SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at Function.parseJSON (jquery.min.js:3)
    at Cwizard.<anonymous> (creation_wizard.js:61)
    at Object.i (jquery.min.js:3)
    at c (jquery.min.js:3)
    at Object.fireWith [as rejectWith] (jquery.min.js:3)
    at k (jquery.min.js:5)
    at XMLHttpRequest.r (jquery.min.js:5)

I assume something is going wrong in the background?
Dovecot and postfix seem to run fine, so do all other components I could test so far.

I did try to re-install modoboa using the install script in debug mode, however except for messages that software and db tables were already existing, I couldn't see a problem. I did try another run after deleting /srv/modoboa, still no change.

Any other solution about similar issues that I found here didn't seem to work, e.g. I don't use a custom logo

# Expected behavior

Creates new  accounts and users

**Comment by tonioo** [comment]
@yodahome Can you set DEBUG to True inside settings.py file, reload Apache and check the response tab in your browser console when you try to create a new account please?

**Comment by yodahome** [comment]
> @yodahome Can you set DEBUG to True inside settings.py file, reload Apache and check the response tab in your browser console when you try to create a new account please?

I did that and it didn't really add any additional useful information to the console. However when going through all the settings I realised that the pdf credentials were activated and apparently couldn't be written to disk because of permission issues (the modoboa user had write access, the apache user www-data did not).
I found a solution to that problem and together with the sudo config to allow modoboa creation of mail folders as the vmail user now user accounts (of any role) are created properly.

This issue can be closed. Thanks!
